src/utils/search/search_tools.py

```
@@ -0,0 +1,132 @@
+
+def get_websearch_tool(user_location: dict[str, Any] | None = None) -> WebSearchTool:
+    """
+    Get OpenAI's WebSearchTool instance with optional location configuration.
+    
+    Args:
+        user_location: Optional location configuration for search results.
+                      Should contain 'country', 'city', 'region', and/or 'timezone'.
+    
+    Returns:
+        Configured WebSearchTool instance for use in Agent configurations.
+    """
```

Ruff (0.11.9)

Multi-line docstring summary should start at the first line

Remove whitespace after opening quotes

(D212)